utils/profile-actions.js

```diff
@@ -4,6 +4,7 @@ import getDocument from './document-generator'
 export function fillEmail(email) {
   cy.get('#cart-to-orderform').click()
   cy.get('#client-pre-email').type(email)
+  cy.wait(3000)
```

Comment:
Why did you change this file if you're not testing profile?

Reply:
I forgot to revert this change when I removed SP tests from the finishing purchase flow.